pages/operations/cluster-configuration.md

```diff
@@ -0,0 +1,171 @@
+---
+title: Cluster Configuration
+description: DevOps workflows that involve editing your cluster Terraform.
```

Comment:
nit: "cluster's Terraform"

Reply:
?

Reply:
fixed, I like cluster's more anyway but either is technically fine